src/bootstrap.js

```diff
@@ -0,0 +1,34 @@
+import has, { add, exists } from '@dojo/framework/has/has';
+import global from '@dojo/framework/shim/global';
+import '@dojo/framework/shim/Promise';
+
+if (!exists('build-time-render')) {
+	add('build-time-render', false, false);
+}
+
+if (global.__public_path__) {
+	__webpack_public_path__ = window.location.origin + global.__public_path__;
+	add('public-path', global.__public_path__, true);
+}
+
+var modules = [];
+
+if (!has('dom-intersection-observer')) {
```

Comment:
i wonder if we should have another set of has flags and check both for excluding these, simply because if someone does a `has()` check elsewhere for say `dom-webanimation` it's going to return true even if the browser doesn't support it, due to us marking it that way in the build to not import it.

Reply:
@matt-gadd That's a good point, perhaps we need something prefixed like `no-bundle-` or `include-`... open to better suggestions, naming things is tough